storage/pipeline/utils.go

```diff
@@ -105,7 +108,18 @@ func simulateMsgGas(ctx context.Context, sa interface {
 		Params: params,
 	}
 
-	return sa.GasEstimateMessageGas(ctx, &msg, nil, types.EmptyTSK)
+	var b bytes.Buffer
+	err := msg.MarshalCBOR(&b)
+	if err != nil {
+		return nil, xerrors.Errorf("failed to unmarshal the signed message: %w", err)
+	}
+	s := hex.EncodeToString(b.Bytes())
+
+	gmsg, err := sa.GasEstimateMessageGas(ctx, &msg, nil, types.EmptyTSK)
+	if err != nil {
+		err = fmt.Errorf("message %s failed: %w", s, err)
```

Comment:
```suggestion

	gmsg, err := sa.GasEstimateMessageGas(ctx, &msg, nil, types.EmptyTSK)
	if err != nil {
		err = fmt.Errorf("message %x failed: %w", b.Bytes(), err)
```

Can log bytes as hex with %x

Reply:
Did not know this. Thanks. Will update it.